src/java_tools/junitrunner/javatests/com/google/testing/junit/runner/model/AntXmlResultWriterTest.java

```diff
@@ -115,6 +116,26 @@ public void testWallTimeAndMonotonicTimestamp() throws Exception {
         .isEqualTo(startTime);
   }
 
+  @Test
+  public void testSkippedOrSuppressedReportedAsSkipped() throws Exception {
+    TestSuiteNode parent = createTestSuite();
+    TestCaseNode skipped = createTestCase(parent);
+    skipped.started(testInstant(Instant.ofEpochMilli(1)));
+    skipped.testSkipped(testInstant(Instant.ofEpochMilli(2)));
+    TestCaseNode suppressed = createTestCase(parent);
+    suppressed.testSuppressed(testInstant(Instant.ofEpochMilli(4)));
+    resultWriter.writeTestSuites(writer, root.getResult());
+    Document document = parseXml(stringWriter.toString());
+    NodeList caseElems = document.getElementsByTagName("testcase");
+    assertThat(caseElems.getLength()).isEqualTo(2);
+    for (int i = 0; i < 2; i++) {
+      Element caseElem = (Element) caseElems.item(i);
+      NodeList skippedElems = caseElem.getElementsByTagName("skipped");
+      System.out.println(stringWriter.toString());
+      assertThat(skippedElems.getLength()).isEqualTo(1);
+    }
+  }
```

Comment:
Please break the test into three blocks: arrange (setting things up), act (running), assert (verifying results). Separate each block with an empty line.
Remove System.out.println.